frontend/src/sass/pages/_tree-guidelines.scss

```diff
@@ -0,0 +1,67 @@
+.wizard-step-header {
+  background: $light-grey;
+  h2 {
+    margin-bottom: 0;
+    margin-top: 0;
+    padding-top: 0.3em;
+  }
+}
+
+.wizard-step-content {
+  background: $white;
+  min-height: 400px;
+  padding-bottom: 1em;
+  padding-top: 1em;
+}
+
+.wizard-step-footer {
+  background: $light-grey;
```

Comment:
Is there any reason an existing color from the https://forest-service-trees-staging.app.cloud.gov/style-guide

Reply:
we are getting rid of wizard related code

Reply:
@lpendem is there a story for that?